dvc/ignore.py

```diff
@@ -201,7 +201,7 @@ def _update_trie(self, dirname: str, trie: PathStringTrie) -> None:
         ignore_file_path = os.path.join(dirname, DvcIgnore.DVCIGNORE_FILE)
         if not matches and self.fs.exists(ignore_file_path):
             new_pattern = DvcIgnorePatterns.from_files(
-                ignore_file_path, self.fs
+                ignore_file_path, self.fs, self.root_dir,
```

Comment:
Not always true for subrepos. Looking into it...

Reply:
For the record: had a discussion with @karajan1001 . Agreed to keep it as is and stabilize it in https://github.com/iterative/dvc/issues/6001